spec/requests/api/public/v1/providers/index_spec.rb

```diff
@@ -0,0 +1,208 @@
+require "rails_helper"
+
+describe "GET public/v1/recruitment_cycle/:recruitment_cycle_year/providers" do
+  let(:organisation) { create(:organisation) }
+  let(:recruitment_cycle) { find_or_create :recruitment_cycle }
+
+  let!(:provider) {
```

Comment:
Potentially use `do end` for multi line blocks?

Reply:
cheers fixed